### PR TITLE
Update flathub.json

### PR DESCRIPTION
**Pull request by aandrew-me**

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: don't remove this.

File: flathub.json [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+    "end-of-life": "Application has been renamed to io.github.aandrew_me.ytdn",
+    "end-of-life-rebase": "io.github.aandrew_me.ytdn",
+    "skip-appstream-check": true
```
Comment (hfiguiere) [inline flathub.json:4]: why did you add this? it's what is causing the build failure.

Comment (aandrew-me) [reply flathub.json:4]: Where should I add the EOL message then?

Comment (aandrew-me) [reply flathub.json:4]: I followed the suggestions from here - https://discourse.flathub.org/t/how-to-change-an-app-id/124

Comment (hfiguiere) [reply flathub.json:4]: LINE 4 is the only line I am commenting about.

Comment (hfiguiere) [reply flathub.json:4]: Here is the documentation

https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

image

At no point it say to add     "skip-appstream-check": true

Comment (aandrew-me) [reply flathub.json:4]: Oh I see, ok let me remove it and add the line which you told me to add.

Comment (aandrew-me) [reply flathub.json:4]: > Here is the documentation
> 
> https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life
> 
> image
> 
> At no point it say to add "skip-appstream-check": true

Yeah I just copy pasted it, nevermind.

Comment (hfiguiere) [reply flathub.json:4]: I told to NOT remove the other. Because you never fixed the no aarch64 build.

**Comment by aandrew-me** [comment]
Ok build was successful in my new pull request

**Pull request closed without merging**